resources/definitions/fdmprinter.def.json

```diff
@@ -4565,7 +4565,17 @@
                     "type": "bool",
                     "default_value": false,
                     "settable_per_mesh": false,
-                    "settable_per_extruder": false
+                    "settable_per_extruder": false,
+                    "children":
+                    {
+                        "smooth_spiralized_contours":
```

Comment:
This was a problem for command line slicing.

Child settings should control no more than a parent setting.

Parent settings are meant as a user-friendly way to edit the underlying real settings which are actually used by the engine.

This commit made command line slicing think there was no magic_spiralzie setting. The engine only uses the leaf settings of the setting tree.